src/styles/cylc/_tree.scss

```diff
@@ -33,12 +33,16 @@ $active-color: #BDD5F7;
 }
 
 .treeitem {
-  display: table;
+  display: block;
   width: 100%;
   .node {
     line-height: 1.8em;
-    display: flex;
-    flex-wrap: nowrap;
+    display: block;
```

Comment:
Making these two `block`, limit the width of the treeitem & its node.

This way, the children elements (like v-list-item's) can be flex, and will have the total width limited by the parent's width.

Using that, we then set `flex-wrap` to `nowrap`. And use the styles from the [Flexbox and Truncated Text article from css-tricks](https://css-tricks.com/flexbox-truncated-text/) to truncate the text.

Reply:
So is it this bit that turns the ellipsis on?

```diff
-<v-layout align-center align-content-center wrap>
+<v-layout align-center align-content-center d-flex flex-wrap>
```